.github/workflows/audit-release.yml

```
@@ -0,0 +1,38 @@
+          fetch-tags: true
+
+      - name: Install Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: "~1.24.4"
+          cache: true
+
+      - name: Audit Release Tags with Base
+        run: go tool stbernard audit -start ${{ inputs.start }} -end ${{ inputs.end }} -base ${{ inputs.base }}
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)